src/applications/vaos/components/calendar/CalendarCheckboxOption.jsx

```diff
@@ -1,29 +1,47 @@
 import React from 'react';
+import classNames from 'classnames';
 
 const CalendarCheckboxOption = ({
-  id,
-  fieldName,
-  value,
   checked,
-  onChange,
+  disabled,
+  fieldName,
+  id,
   label,
-}) => (
-  <div className="vaos-calendar__option">
-    <input
-      id={`checkbox-${id}`}
-      type="checkbox"
-      name={fieldName}
-      value={value}
-      checked={checked}
-      onChange={onChange}
-    />
-    <label
-      className="vads-u-margin--0 vads-u-font-weight--bold vads-u-color--primary"
-      htmlFor={`checkbox-${id}`}
-    >
-      {label}
-    </label>
-  </div>
-);
+  onChange,
+  value,
+}) => {
+  const divClasses = classNames('vaos-calendar__option', {
+    disabled,
+  });
+
+  const labelClasses = classNames(
+    'vads-u-margin--0',
+    'vads-u-font-weight--bold',
+    {
+      'vads-u-color--primary': !disabled,
+      'vads-u-color--gray-lighter': disabled,
+    },
+  );
+
+  return (
+    <div className={divClasses}>
+      <input
+        id={`checkbox-${id}`}
+        type="checkbox"
+        name={fieldName}
+        value={value}
+        checked={checked}
+        onClick={onChange}
+        onChange={onChange}
+      />
+      <label className={labelClasses} htmlFor={`checkbox-${id}`}>
+        <span aria-hidden="true">{label}</span>
+        <span className="vads-u-visibility--screen-reader">
+          {label} option selected
```

Comment:
This label was confusing when I listened to it on a screen reader. What I heard was: "AM option selected, checkbox not checked" or "AM option selected, checkbox checked". I think it'd be worth reaching out to a content specialist for the right language. Maybe something like "AM appointment" which would then read "AM appointment, checkbox (un)checked"

/cc @peggygannon